config/resources/besluit-domain-en.lisp

```diff
@@ -84,8 +84,8 @@
                 (:start-date :date ,(s-prefix "mandaat:bindingStart")))
   :has-one `((administrative-unit :via ,(s-prefix "besluit:bestuurt")
                                   :as "administrative-unit")
-  ;;            (bestuursorgaan-classificatie-code :via ,(s-prefix "besluit:classificatie")
-  ;;                                               :as "classificatie")
+             (bestuursorgaan-classificatie-code :via ,(s-prefix "besluit:classificatie")
```

Comment:
@LBONTEN **Blocker:** Can you translate `governing-body-classification-code` and `classification`

Reply:
👍